docs/duration.md

```diff
@@ -73,11 +73,11 @@ new Temporal.Duration(); // => PT0S
 
 ## Static methods
 
-### Temporal.Duration.**from**(_thing_: any) : Temporal.Duration
+### Temporal.Duration.**from**(_durationLike_: Temporal.Duration | object | string) : Temporal.Duration
 
 **Parameters:**
 
-- `thing`: A `Duration`-like object or a string from which to create a `Temporal.Duration`.
+- `durationLike`: a value convertible to a `Temporal.Duration`.
```

Comment:
These "_${type}Like_" changes don't seem quite right to me; I would say that (w.l.o.g.) "duration-like" refers to just an **object** having the right properties (analogous to "[array-likes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array#array-like_objects)" having `length` and array index properties). Generically, I guess the input is "_durationable_"/"_instantable_"/"_datetimeable_"/etc., although such names derived from already compound foundations aren't great. Maybe we should just tweak the "_thing_" convention and name them all "_item_" or "_input_" or "_coercible_" (and probably update the [spec](https://tc39.es/proposal-temporal/#sec-temporal.plaindate.from) to match)?

Reply:
Sure, if _durationLike_ suggests an object then it's probably less confusing to use a vaguer term such as _item_.